packages/web-frontend/cypress/scripts/testE2e/pushSnapshots.js

```diff
@@ -9,7 +9,7 @@ import { SNAPSHOTS } from '../../constants';
 
 export const pushSnapshots = async (snapshotRepo, branch, snapshots) => {
   const date = Date.now();
-  const updateBranch = `tupaiabot/snapshot-update/${branch}_${date}`;
+  const updateBranch = `beyondessential-bot/snapshot-update/${branch}_${date}`;
```

Comment:
@edmofro I sneakily renamed the bots in this "minor fixes" commit while rebasing